stdlib/universe/sort2_test.flux

```diff
@@ -0,0 +1,41 @@
+package universe_test
+
+
+import "csv"
+import "testing"
+
+option now = () => 2030-01-01T00:00:00Z
+
+testcase sort_panic {
```

Comment:
Can you rename this to something that describes what is being tested? Maybe something like `sort_with_null_columns`?